docs/installingLocal.md

```diff
@@ -83,3 +83,21 @@ cd /PrairieLearn
 ./docker/test_js.sh
 ./docker/test_python.sh
 ```
+
+### Connecting to an Existing Docker
+
+The previous shells were launched in their own containers. If you want to open a shell in a Docker container that is *already running*, you can find the container's name and connect to it.
+
+1. Find the name of your running PrairieLearn container:
+
+```sh
+docker ps --format '{{.Image}}: {{.Names}}'
+```
+
+This will output your running container(s) in the form `<container_image>: <container_name>`. Find the `prairielearn/prairielearn: <container_name>` line and copy its `<container_name>` (e.g., `adoring_gauss`).
+
+2. Open a shell in `<container_name>`:
+
+```sh
+docker exec -it <container_name> /bin/bash
+```
```

Comment:
### Using tmux in a container

The PrairieLearn Docker images are built with `tmux` installed. If you start a container with a shell then you can first run `tmux` before running other commands.

Tmux creates virtual windows which run simultaneously (you only see one window at a time). Tmux is controlled by typing a `Ctrl-b` and then another key. The basic commands are:

* `Ctrl-b` `c` - create a new window
* `Ctrl-b` `0` - switch to window number 0 (also `Ctrl-b` `1` switches to window 1, etc.)
* `Ctrl-b` `d` - detaches from tmux back to the original shell, which you can exit to terminate the container

Google `tmux` for tutorials that demonstrate many more capabilities.

Reply:
Sorry, I see now that I wrote this as a comment, I meant to propose it as a change.
=

Reply:
@mwest1066 fixed that for you.

Future reference, add an addition backtick, e.g. 

`````md
Extra backtick
````suggestion

```
Documentation
```

````

Reply:
Ahhhh! Many thanks.